src/mpl/include/mpl_atomic_by_lock.h

```diff
@@ -12,128 +12,133 @@
 {                                                                              \
     TYPE prev;                                                                 \
-    MPL_ATOMIC_IPC_SINGLE_CS_ENTER();                                          \
+    ATOMIC_IPC_SINGLE_CS_ENTER();                                              \
     prev = (TYPE)ptr->v;                                                       \
     ptr->v = val;                                                              \
-    MPL_ATOMIC_IPC_SINGLE_CS_EXIT();                                           \
+    ATOMIC_IPC_SINGLE_CS_EXIT();                                               \
     return prev;                                                               \
 }
 
-#define MPL_ATOMIC_DECL_FUNC_FAA(TYPE, NAME)                                   \
+#define ATOMIC_DECL_FUNC_FAA(TYPE, NAME)                                       \
 static inline TYPE MPL_atomic_fetch_add_ ## NAME                               \
                             (struct MPL_atomic_ ## NAME ## _t * ptr, TYPE val) \
 {                                                                              \
     TYPE prev;                                                                 \
-    MPL_ATOMIC_IPC_SINGLE_CS_ENTER();                                          \
+    ATOMIC_IPC_SINGLE_CS_ENTER();                                              \
     prev = (TYPE)ptr->v;                                                       \
     ptr->v += val;                                                             \
-    MPL_ATOMIC_IPC_SINGLE_CS_EXIT();                                           \
+    ATOMIC_IPC_SINGLE_CS_EXIT();                                               \
     return prev;                                                               \
 }                                                                              \
 static inline TYPE MPL_atomic_fetch_sub_ ## NAME                               \
                             (struct MPL_atomic_ ## NAME ## _t * ptr, TYPE val) \
 {                                                                              \
     TYPE prev;                                                                 \
-    MPL_ATOMIC_IPC_SINGLE_CS_ENTER();                                          \
+    ATOMIC_IPC_SINGLE_CS_ENTER();                                              \
     prev = (TYPE)ptr->v;                                                       \
     ptr->v -= val;                                                             \
-    MPL_ATOMIC_IPC_SINGLE_CS_EXIT();                                           \
+    ATOMIC_IPC_SINGLE_CS_EXIT();                                               \
     return prev;                                                               \
 }
 
-#define MPL_ATOMIC_DECL_FUNC_VAL(TYPE, NAME) \
-        MPL_ATOMIC_DECL_FUNC_COMMON(TYPE, NAME) \
-        MPL_ATOMIC_DECL_FUNC_FAA(TYPE, NAME)
+#define ATOMIC_DECL_FUNC_VAL(TYPE, NAME) \
```

Comment:
Since they are in header files and exposed to all units, recommend add `MPLI_` prefix.

Reply:
I will add `MPLI_`.